core/git_mixins/history.py

```diff
@@ -63,12 +65,13 @@ def log(self, author=None, branch=None, file_path=None, start_end=None, cherry=N
 
         return entries
 
-    def commit_generator(self, limit = 6000):
+    def commit_generator(self, limit = 6000, follow=False):
```

Comment:
This is related to an earlier refactoring, should we actually pass the arguments `branch` and `file_path`?